docs/proposals/resize-when-creating-by-group.md

```diff
@@ -0,0 +1,127 @@
+## Summary
+
+Expand the PVC's initial capacity based on the largest capacity in specified PVCs.
+
+### Motivation
+
+One use case for pvc-autoresizer is automatically extending a PV/PVC for multiple Pod/PV pairings, such as a MySQL cluster.
+In such a cluster, if a Pod/PV fails and a PV/PVC is rebuilt for the restore process, a PV is created based on the PVC template. Even if a PV of the same size as the others is actually needed, a PV of the template's capacity is created, resulting in a case where the restore process stops due to insufficient capacity.
+
+To solve this problem, we are planning to add a new function to pvc-autoresizer to create a PV/PVC with sufficient capacity at the time of creation, when a PVC of the same group already exists.
+
+### Goals
+
+- Provide the ability to expand the PVC's initial capacity based on the largest capacity in specified PVCs.
+- The PVC for determining capacity should be user-selectable.
+- [moco](https://github.com/cybozu-go/moco) checks the PV capacity at the beginning of the restore process. It should be able to handle such cases.
+
+## Proposal
+
+Rules for grouping & initial sizing
+
+- If creating a PVC has a `resize.topolvm.io/pre-resize-group-by` annotation and the label specified in the annotation exists, existing PVCs with matching values are determined to be in the same group.
```

Comment:
IMHO, the name `resize.topolvm.io/pre-resize-group-by` is ambiguous because it is not clear what pre refers to.

Reply:
@peng225 @daichimukai @satoru-takeuchi 

How about `resize.topolvm.io/init-resize-group-by` or `resize.topolvm.io/initial-resize-group-by` ?

Reply:
I prefer `resize.topolvm.io/initial-resize-group-by`.

Reply:
LGTM

Reply:
I agree with @peng225 and @daichimukai